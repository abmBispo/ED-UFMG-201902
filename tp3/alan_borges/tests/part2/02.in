8
na sala na mesma sala sentada na marquesa
c na
c sala
c sentada
c marquesa
c mesma
