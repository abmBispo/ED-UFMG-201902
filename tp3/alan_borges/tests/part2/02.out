11
01
00
100
101
